docs/cmd/kn_service_update.md

```diff
@@ -27,6 +27,7 @@ kn service update NAME [flags]
 ### Options
 
 ```
+      --async                    Update service and don't wait for it to become ready.
```

Comment:
we should rename this flag to --wait=bool. async is too confusing imho with serveless "async" handler configuration (no expected response).

Reply:
Happy to rename when we decided to on the general direction. See #283 and #284 for voting how to continue with boolean flags (like `--help=true`)